src/arch/arm/mmu/mmu_small_page.c

```diff
@@ -124,7 +126,9 @@ uint32_t vmem_info(uint32_t vaddr)
 
     vaddr &= ~0xFFF;
 
-    printk("vmem info: base %p; vaddr %p", (void*) vmem_current_context(), (void *) vaddr);
+    printk("vmem info: base %p; vaddr %p",
```

Comment:
log_xxx?

Reply:
This function is not used directly right now, so I don't think it should me modified this way.
May be it will be used in some command, so `printf()` will be more appropriate.